samples/snapshot/pom.xml

```diff
@@ -28,7 +28,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-logging</artifactId>
-      <version>3.1.3-SNAPSHOT</version>
+      <version>3.2.0</version>
```

Comment:
Should the version here to keep `-SNAPSHOT` suffix? Otherwise, it is a problem to run samples on the snapshot which is the purpose of this pom.xml